### DISCUSSION TITLE
Can i connect 2 bilge pumps to SH-ESP32

### DISCUSSION THREAD

**Discussion by big-col** [hatlabs/discussions#104] [General]
2 part question.  In the SENSESP Getting Started page it mentions:  

Connect a piece of jumper wire from a 3.3V pin to the GPIO pin 36 on the ESP32 device. (On SH-ESP32 boards, the pin is labeled VP.) You should see the analog input voltage jump to close to 3.3V.

Next, connect the jumper wire between GND and GPIO pin 13. This should make the digital input 2 value change to false.

I want to connect the Bilge alarm LED to monitor bilge pump activations. Do i connect this to pin 13 on a SH-ESP?

Second question is can i connect a 2nd bilge pump to another pin?

Is there any limitations to voltages connecting to pin 13 or a second pin?

Thanks

**Comment by mairas** [comment]
If you know your input always stays between 0 and 3.3V, you can connect them directly to any of the free GPIO pins listed here: https://docs.hatlabs.fi/sh-esp32/pages/hardware/#pinouts-of-peripherals. It's a bit risky, though, because the pins are connected the ESP32 MCU without any protection. Just so that you're aware. Using the optocoupler you asked about in the other post is a bullet-proof approach.

If you're not using I2C, one handy trick is using the I2C SDA and SCL pins for input; they have some noise and ESD protection. The same 0..3.3V voltage range limitation still applies.

**Comment by big-col** [comment]
I'm thinking the safest thing to do in this situation is use a Halmet because I'm not 100% in the voltage always being less than 3.3v
